bin/pytorch_inference/CCommandParser.cc

```diff
@@ -155,7 +147,7 @@ bool CCommandParser::validateJson(const rapidjson::Document& doc,
     return true;
 }
 
-bool CCommandParser::checkArrayContainsUInts(const rapidjson::Value& arr) const {
```

Comment:
nit: these methods are really one liners and would be ever so slightly more efficient since `find_if` returns false and stops traversing on failure
```
return std::find_if(arr.Begin(), arr.End(), [](const auto& i) { return itr->IsUint64() == false; }) == arr.End();
```